snap/README.md

```diff
@@ -290,32 +297,23 @@ JWT_SIGNATURE=`echo -n "$JWT_HEADER.$JWT_PAYLOAD" | openssl dgst -sha256 -binary
 TOKEN=$JWT_HEADER.$JWT_PAYLOAD.$JWT_SIGNATURE
 ```
 
-The resulting JWT token must be included
+4. Once you have the token you can access the API Gateway as follows:
+
+The JWT token must be included
 via an HTTP `Authorization: Bearer <access-token>` header on any REST calls used to access EdgeX services via the API Gateway. 
 
 Example:
 
 ```bash
-$ curl -k -X GET https://localhost:8443/coredata/api/v1/ping? -H "Authorization: Bearer $TOKEN"
+$ curl -k -X GET https://localhost:8443/core-data/api/v2/ping? -H "Authorization: Bearer $TOKEN"
 ```
 
-Additional users can be added by repeatedly calling the secrets-config command as above. Only one user can however be set at any time when using the snap configure hook, so
-the current user must first be removed by setting the snap configuration settings to an empty string, before setting the values again:
-
-```bash
-$ sudo snap set edgexfoundry env.security-proxy.user=""
-$ sudo snap set edgexfoundry env.security-proxy.public-key=""
-$ sudo snap set edgexfoundry env.security-proxy.user=user02,USER_ID2,ES256
-$ sudo snap set edgexfoundry env.security-proxy.public-key="$(cat public.pem)"
-```
+Additional users can be added by repeatedly calling the secrets-config command as above.  
   
 
 #### API Gateway TLS certificate setup
 
-By default Kong is configured with an EdgeX signed TLS certificate. Client validation of this certificate requires the root CA certificate from the EdgeX instance. This file
-(`ca.pem`) can be copied from directory `$SNAP_DATA/secrets/ca`.
-
-It is also possible to install your own TLS certificate to be used by the gateway. The steps to do so are as follows:
+By default Kong is configured with a selfsigned TLS certificate. It is also possible to install your own TLS certificate to be used by the gateway. The steps to do so are as follows:
```

Comment:
Is that possible to include this TLS certificate's file path?

Reply:
~selfsigned~ self-signed  TLS certificate

Reply:
yes, added that